osu.Game/Overlays/Chat/Tabs/ChannelSelectorTabItem.cs

```diff
@@ -39,5 +40,7 @@ public ChannelSelectorTabChannel()
                 Name = "+";
             }
         }
+
+        protected override bool OnMouseUp(MouseUpEvent e) => false;
```

Comment:
Why is it overriding this?

Reply:
Is this unnecessary?
The "+"-Button tab should not be closable.

Reply:
Ah I see.

Reply:
It's not required. THere's a check (as you'd expect) to disallow closing in the first place.